src/include/cell_inline.h

```diff
@@ -1635,8 +1633,7 @@ __wt_page_cell_data_ref_kv(
         uint8_t *__cell;                                                                        \
         for (__cell = WT_PAGE_HEADER_BYTE(S2BT(session), dsk), __i = (dsk)->u.entries; __i > 0; \
              --__i) {                                                                           \
-            __wt_cell_unpack_delta_int(                                                         \
-              session, page_dsk, dsk, (WT_DELTA_CELL_INT *)__cell, &(unpack));                  \
+            __wt_cell_unpack_delta_int(session, page_dsk, dsk, (WT_CELL *)__cell, &(unpack));   \
```

Comment:
This is still not changed?

Reply:
Fixed in f334c4666fc996d956f8c53444edc0a913e58079